Added command to get resources from validation manifest

meganwolf0 opened the pull request:
Added `dev get-resources` command to Lula. Need to specify and input file (-f) for the validation manifest and optionally an output file (-o) for the resources json data.

Tested with a local k3d with podinfo deployed and the following validation.yaml input:
```yaml
target:
  provider: opa
  domain: kubernetes
  payload:
    resources:
      - name: deployment
        resource-rule:
          name: podinfo
          group: apps
          version: v1
          resource: deployments
          namespaces: [podinfo]
    rego: |
      package validate
      
      default validate = false
```

meganwolf0 commented:
@brandtkeller - threw this together real quick after our discussion, probably needs more testing and whatnot but would love any feedback on if this is hitting what you were thinking

brandtkeller commented:
> @brandtkeller - threw this together real quick after our discussion, probably needs more testing and whatnot but would love any feedback on if this is hitting what you were thinking

@meganwolf0 Looks great! E2E test here is likely most applicable. I'd checkout whether `message.JSONValue()` provides any value otherwise this meets the intent.

meganwolf0 commented:
Added some e2e testing and refactored the get-resources command a bit. I also made a change to the resources that get returned and dropped the `metadata.managedFields`. This field seems kind of useless and introduces a lot of lines and potentially noise so I figured it'd be ok to drop. If there's an issue or maybe more discussion is needed I could always revert this and throw it into a separate PR.

brandtkeller commented on src/cmd/dev/get-resources.go:
```diff
@@ -0,0 +1,117 @@
+
+type flags struct {
+	InputFile  string // -f --input-file
+	OutputFile string // -r --result-file
```
minor copy/paste in the field comments (`-r` vs intended `-o`)

meganwolf0 replied:
oops should be fixed!
